### ISSUE TITLE
Server Response Time Chart rendering incorrect data with version 2.2.0

### ISSUE THREAD

**Issue by vinnieking06** [microsoft/ApplicationInsights-node.js#882]
I upgraded from 1.8.10 to 2.2.0 and then in the Application Insights Dashboard overview, our server responses times dropped to 1ms. They normally are about 200ms and after the upgrade, they showing about 1ms. 

Here is the 6 hour view. The upgrade to 2.2.0 was deployed around 1pm:
![Screen Shot 2021-12-17 at 2 28 20 PM](https://user-images.githubusercontent.com/18544837/146610053-0fad3fa6-22a8-401a-9a62-72c21a65d061.png)

1 hour view:
![Screen Shot 2021-12-17 at 2 27 49 PM](https://user-images.githubusercontent.com/18544837/146609998-c94add2f-92af-4ef1-98c8-b2c25c7c7371.png)

**Comment by hectorhdzg** [comment]
@vinnieking06 is this still happening on your side?, are you sampling any data?, if you are not please disabled pre aggregated metrics to ensure this is not causing the issue, you can use setAutoCollectPreAggregatedMetrics(false) more info [here](https://github.com/microsoft/ApplicationInsights-node.js#configuration), please let us know the results.

**Comment by vinnieking06** [comment]
@hectorhdzg Yes it's still happening, and yes we are sampling. Here are out sampling settings: 
appInsights.defaultClient.config.samplingPercentage = 33;
Since we are sampling 33%, do you still recommend trying to disable pre aggregated metrics?

**Comment by vinnieking06** [comment]
@hectorhdzg A response time of 0-2ms would be impossible for our app, since we make a call an api that takes at least 90ms. Also, when you click on the performance graph and it goes to full screen, and then refresh the data, the data looks good, as is shown here:
![Screen Shot 2022-01-26 at 11 31 06 AM](https://user-images.githubusercontent.com/18544837/151224658-47ce45ce-06e2-4e65-8777-705d5f765a11.png)

**Comment by hectorhdzg** [comment]
@vinnieking06 can you query your Request data and take a look at the duration in there? I would like to understand where the incorrect data could be generated. Also are you using multiple TelemetryClients in your app?

**Comment by vinnieking06** [comment]
@hectorhdzg, it looks good when I query it. Low 100ms, which is what we expect.
![Screen Shot 2022-01-27 at 2 37 37 PM](https://user-images.githubusercontent.com/18544837/151447811-a5bf22c1-097c-44da-8f68-b378ac02d984.png)

**Comment by vinnieking06** [comment]
Here is the raw data. Most are in the 100 range, but I did see a couple that were around 40 and one that was like 1ms.
![Screen Shot 2022-01-27 at 2 41 35 PM](https://user-images.githubusercontent.com/18544837/151448343-6147431d-69b5-4cc4-91c9-be5ac8f52d49.png)

**Comment by vinnieking06** [comment]
@hectorhdzg As for using multiple TelemtryClients, yes, we also send logs to splunk.
